### PR TITLE
add blog

### LINKED ISSUE #1: Create a blog hosted on github
## Scope

* neume requires somewhere to host external writing about the project
* The initial blog post should be available alongside the rebranding of this Github organisation from musicOS to neume, to provide context around the reasons for the rename and positioning of each project; neume being the future name for the open source data crawling and indexing work, musicOS being the front end application being built by HIFI Labs
* The solution should be something that is easy to push writing out to by the core team and can be addressed through the neume TLD (neume.network)
* [Hexo](https://hexo.io/) has been mentioned as an option for framework, alongside other options such as [Jekyll](https://jekyllrb.com/); other options can also be considered

## Notes

* Discussion on the proposed approach should be conducted within this Issue, through to sign-off from myself (@djfnd) and @reimertz 

## Deliverables

* A functioning blog interface that is hosted within this GitHub, updatable by the team, and publicly accessible

## Deadline

* Go live by 18th May

### PR DESCRIPTION
**Pull request by il3ven**
This PR adds blog functionality using 11ty and EleventyDuo theme. Live demo at https://il3ven.github.io/neume-network-website. Fixes #1.

#### Fix for GitHub Page
The theme is now deployable to Github pages. Github hosts pages at `user.github.io/repo` so links break. For example a link to `/posts` will point to `user.github.io/posts` instead of `user.github.io/repo/posts`. This was one of the fix.

#### Comment unnecessary code
I have commented out [mermaid](https://www.npmjs.com/package/mermaid) which is a diagramming tool. This reduces bundle size by 300kb. (Lighthouse scores are now 🚀)

#### Personalise Blog
I have personalised the blog i.e mention neume-network and change social links. `src/data/site.json` contains many constants.

#### Add workflow to deploy to Github Pages
I have added a workflow to deploy to gh pages. In order for Github pages to work on neume-network/website repo there are a few steps:
- Enable Github pages from settings
- Select gh-pages as the branch and root as the directory
- Create a gh-pages manually. The workflow doesn't have permission to create new branches. It can only update.

I can do the above steps but I will need access to the repo.

### Further Changes

We can still personalise it more. Change the text on the welcome or about page. We can even add new pages or change some css styling. Let me know if any further changes are required. We can open new issues for that. Though in the current state the blog does good. Anymore changes will be subjective.

### TODO

- [x] Fix sitemap
- [x] Change font
- [x] Add readme on how to add new posts
- [x] Incorporate plausible

### CODE REVIEW

**Review by TimDaub — CHANGES_REQUESTED** [review]
- It looks clean, and also I like that the page has a small footprint :)
- A minor legal annoyance: For the EU, Google Fonts isn't GDPR compliant by default: https://usercentrics.com/knowledge-hub/google-fonts-gdpr-compliant/
  - I recommend finding either a web-safe font similar to what we're already using or vendoring in the font as shown in the article above.
  - Alternative is including a cookie banner (which I'd like to avoid)

**Comment by il3ven** [comment]
> A minor legal annoyance: For the EU, Google Fonts isn't GDPR compliant by default.

I'll look into this. I didn't know about Google fonts and GDPR. Which font are we currently using?

I also found that the sitemap isn't working. This [problem](https://eleventyduo.netlify.app/sitemap.xml) is present in the theme too. I'll look into this too.

File: src/layouts/base.njk [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="canonical" href={{ page.url | url | absoluteUrl(site.url) }}>
+
+    {% bundledcss %}
+
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans&display=swap" rel="stylesheet">
```
Comment (TimDaub) [inline src/layouts/base.njk]: replace bc of gdpr

File: src/css/typography.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  font-size: var(--text-lg);
+  line-height: 1.54;
+  color: var(--color);
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  font-feature-settings: 'liga', 'tnum', 'case', 'calt', 'zero', 'ss01', 'locl';
+  font-variant-ligatures: contextual;
+  -webkit-overflow-scrolling: touch;
+  -webkit-text-size-adjust: 100%;
+  font-family: 'IBM Plex Sans', sans-serif;
```
Comment (TimDaub) [inline src/css/typography.css]: replace bc of gdpr

**Comment by reimertz** [comment]
Thanks @TimDaub for bringing up not using Google fonts. 

On that note,I have setup a Cloudflare worker to enable a [Plausible.io](https://plausible.io) proxy for a privacy-first, GDPR-complaint cookieless analytics.

All we have to do is to add a script tag with `https://neume.network/neume/js/info.js` and everything else should work as intended.

```
<script defer data-domain="neume.network" src="https://neume.network/neume/js/info.js"></script>
```

**Comment by TimDaub** [comment]
> On that note,I have setup a Cloudflare worker to enable a [Plausible.io](https://plausible.io) proxy for a privacy-first, GDPR-complaint cookieless analytics.

That's awesome! Thanks!

**Comment by reimertz** [comment]
I took the liberty of connecting this repo to Netlify for smooth preview deploy integration. So hopefully, the next commit will generate a preview link. 

Once the site is deployed, I'll help setting up Cloudflare cache purge so the site can be proxied by Cloudflare CDN with aggressive cache, but then purged once new deploys happen.

Lmk if this sounds good, if not, I can revert the integration. :)

**Comment by reimertz** [comment]
Netlify deploy preview seems to be working smoothly now.

**Comment by djfnd** [comment]
Apologies, I missed the PR previously.
Font-wise, we are running under the assumption of Helvetica for the design work that is ongoing, if that helps.

**Review by TimDaub — APPROVED** [review]
great job @il3ven!

**Pull request merged**